storage/ndb/src/kernel/vm/mt.hpp

```diff
@@ -48,10 +48,7 @@
   messages directed to other nodes and contains no blocks and
   executes thus no signals.
 */
-#define MAX_BLOCK_THREADS (MAX_MAIN_THREADS +       \
-                           MAX_NDBMT_LQH_THREADS +  \
-                           MAX_NDBMT_TC_THREADS +   \
-                           MAX_NDBMT_RECEIVE_THREADS)
+#define MAX_BLOCK_THREADS (NDBMT_MAX_BLOCK_INSTANCES)
```

Comment:
NDBMT_MAX_BLOCK_INSTANCES = 722;

the previous value of MAX_BLOCK_THREADS is defined as 2+332+160+80 = 574

Reply:
The biggest machines at our disposal is actually 360 VCPUs, so I set the limit to more or less twice that but taking 2 main threads into account. This limit isn't really tested, so at some point in time we should probably ensure that it actually works to create such big nodes, so for the moment it is a bit of a theoretical limit.